### PR TITLE
Change how user is accessed to avoid current_user

### PR DESCRIPTION
**Pull request by jessebrennan**
When requesting the endpoint /authorization from azul-webservice current_user would be anonymous (despite the cookie and session details being sent along with the request. The current work-around ensures that the cookie is checked for the user info regardless.

This is currently deployed on jesse.ucsc-cgp-dev.org

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -441,6 +464,14 @@ def authorization():
     User is authorized, return 204
     User is not authorized, return 403
     """
+    # For a yet unsolved reason
+    # current user is anonymous when called from azul-webservice, even though the
+    # session cookie is sent over. overwriting the anonymous user with User() ensures
+    # that the session cookie is checked for user info and cookies
+    user = get_user()
+
+    print 'request headers:', request.headers
```
Comment (coverbeck) [inline app.py:473]: I don't think you should print this, nor the next line. Especially if Auth header had a token.

File: app.py [hunk 2]
```diff
@@ -451,7 +482,12 @@ def authorization():
     else:
         if bearer != "Bearer":
             return "Authorization must start with Bearer", 401
-    if auth_token is None and current_user.is_anonymous:
+    #FIXME current user doesnt work!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```
Comment (coverbeck) [inline app.py:485]: Do you want to remove these prints as well?

**Review by coverbeck — CHANGES_REQUESTED** [review]
This is going to break session protection, but not sure how else to handle it.

**Comment by jessebrennan** [comment]
It turns out that this mechanism is bad since it undermines session protection. Session protection will still work if the request from azul-webservice is  formatted with the correct headers.

**Pull request closed without merging**